### PR TITLE
Functional Tests : BO - Catalog - Products : Pack Tab

### PR DESCRIPTION
**Pull request by Progi1984**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Functional Tests : BO - Catalog - Products : Pack Tab
| Type?             | improvement
| Category?         | TE
| BC breaks?        | no
| Deprecations?     | no
| How to test?      | CI is :green_circle: 
| UI Tests          | https://github.com/Progi1984/ga.tests.ui.pr/actions/runs/6223375149
| Fixed issue or discussion?     | N/A
| Related PRs       | N/A
| Sponsor company   | @PrestaShopCorp

### CODE REVIEW

File: tests/UI/pages/BO/catalog/productsV2/add/index.ts [hunk 1]
```diff
@@ -211,7 +211,9 @@ class CreateProduct extends BOBasePage {
       priceTaxExc: await this.getTextContent(page, this.productHeaderTaxExcluded),
       priceTaxIncl: await this.getTextContent(page, this.productHeaderTaxIncluded),
       quantity: await this.getTextContent(page, this.productHeaderQuantity, false),
-      reference: await this.getTextContent(page, this.productHeaderReference('reference'), false),
+      reference: (await page.locator(this.productHeaderReference('reference')).count())
+        ? await this.getTextContent(page, this.productHeaderReference('reference'), false)
+        : '',
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/catalog/productsV2/add/index.ts:216]: Maybe create specific functions for this one and others, so we could use individually ?

File: tests/UI/pages/BO/catalog/productsV2/index.ts [hunk 2]
```diff
@@ -304,8 +304,8 @@ class Products extends BOBasePage {
     await this.waitForVisibleSelector(page, `${this.modalCreateProduct} iframe`);
     await this.waitForHiddenSelector(page, this.modalCreateProductLoader);
 
-    const createProductFrame: Frame | null = await page.frame({url: /sell\/catalog\/products-v2\/create/gmi});
-    await expect(createProductFrame).to.be.not.null;
+    const createProductFrame: Frame | null = page.frame({url: /sell\/catalog\/products-v2\/create/gmi});
+    expect(createProductFrame).to.be.not.eq(null);
```
Comment (boubkerbribri) [inline tests/UI/pages/BO/catalog/productsV2/index.ts:308]: expect should be removed from pages (I think it's a WIP on delete productV1)

**Review by boubkerbribri — APPROVED** [review]
2 comments but LGTM

**Review by nesrineabdmouleh — APPROVED** [review]

**Comment by nesrineabdmouleh** [comment]
Thank you @Progi1984

**Pull request merged**